cabal-install/changelog

```diff
@@ -1,6 +1,8 @@
 -*-change-log-*-
 
 2.4.0.0 (current development version)
+	* Add 'new-sdist' command (#5389). Creates stable archives based on
```

Comment:
Please also add a note to the Cabal changelog saying that sdist hooks are now deprecated and `autogen-modules` and `build-tool-depends` should be used instead.